mpress-custom-front-page.php

```diff
@@ -30,7 +30,7 @@ class mPress_Custom_Front_Page {
 		 * @return mPress_Custom_Front_Page
 		 */
 		public static function get_instance() {
-			return self::$instance ?? new self();
+			return isset( self::$instance ) ? self::$instance : new self();
```

Comment:
My IDE converted this to PHP 7.x coalescing comparison. I reverted it back to PHP 5.x